gprofiler/databricks_client.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright (c) Granulate. All rights reserved.
+# Licensed under the AGPL3 License. See LICENSE.md in the project root for license information.
+#
+
+import json
+from typing import Dict, Optional
+
+import requests
+
+from gprofiler.log import get_logger_adapter
+
+HOST_KEY_NAME = "sink.ganglia.host"
```

Comment:
ganglia is an optional feature for Spark as I understand it. Is it mandatory on DBX?

Reply:
It seems so:

> To access the Ganglia UI, navigate to the Metrics tab on the cluster details page. **CPU metrics are available in the Ganglia UI for all Databricks runtimes**.

https://docs.databricks.com/clusters/clusters-manage.html#ganglia-metrics

Reply:
kk